src/builder.rs

```diff
@@ -667,6 +667,11 @@ impl Builder {
     /// if the leftmost bit was one.
     ///
     /// ```rust,no_run
+	/// //fix doc error about overflowing_literals
+	/// //rendered rfc: https://github.com/rust-lang/rfcs/blob/master/text/2438-deny-integer-literal-overflow-lint.md
+	/// //tracking issue: https://github.com/rust-lang/rust/issues/54502
```

Comment:
This should be prefixed by 4 spaces, not a tab

Reply:
I have fixed this problem, but the CI test crashes again, please restart the test tasks again. Also, is there any way I can restart these test tasks myself?